src/System.Net.Requests/ref/project.json

```diff
@@ -1,15 +1,15 @@
 {
   "dependencies": {
-    "System.IO": "4.0.0",
-    "System.Net.Primitives": "4.0.10",
-    "System.Net.WebHeaderCollection": "4.0.0",
-    "System.Runtime": "4.0.0",
-    "System.Threading.Tasks": "4.0.0"
+    "System.IO": "4.3.0-beta-24522-03",
+    "System.Net.Primitives": "4.3.0-beta-24522-03",
+    "System.Net.WebHeaderCollection": "4.3.0-beta-24522-03",
+    "System.Runtime": "4.3.0-beta-24522-03",
+    "System.Threading.Tasks": "4.3.0-beta-24522-03"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.7": {
       "imports": [
```

Comment:
nit: These imports are no longer need when consuming our latest packages and can be removed.